tests/manage/mcg/test_object_integrity.py

```diff
@@ -0,0 +1,90 @@
+import logging
+
+import boto3
+import pytest
+
+from ocs_ci.framework import config
+from ocs_ci.framework.testlib import ManageTest, tier1
+from ocs_ci.ocs import constants
+from tests.helpers import craft_s3_command
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.mark.skipif(
+    condition=config.ENV_DATA['platform'] != 'AWS',
+    reason="Tests are not running on AWS deployed cluster"
+)
+def check_md5sum(original_object, result_object, awscli_pod):
```

Comment:
isn't it something that will be needed for other tests as well?
If so, please move to helpers